relay-config/src/config.rs

```diff
@@ -524,13 +524,17 @@ struct SentryMetrics {
     ///
     /// Defaults to 5.
     pub meta_locations_max: usize,
+    /// Whether to override the [`received_at`] field in the [`BucketMetadata`] with the current
+    /// receive time of the instance.
+    pub override_received_at_metadata: bool,
```

Comment:
In production, we would like to set this option to true only for pop relays, since we want to have the `received_at` metadata field set to the outermost relay timestamp that receives a given bucket.

Reply:
We already have that handling in the processor as `keep_metadata` for example in `handle_process_metrics` which should already do the right thing.